components/10grid.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/docs.css">
+    <title>Even UI | Grid</title>
+    <script src="https://kit.fontawesome.com/9f366bbf8a.js" crossorigin="anonymous"></script>
+    <script src="./index.js"></script>
+</head>
+
+<body>
+    <nav class="topnav" id="myTopnav">
```

Comment:
you can easily target navbar with `document.getElementByClassName('topnav')[0]` , no need of using separate id as it will decrease readability